enterprise/utils.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to be used across the enterprise djangoapp
```

Comment:
@haikuginger suggestion - this `utils` file only contains parts relevant to course catalogue so far. I would suggest renaming it to `course_catalogue_api` and keeping it dedicated to course catalogue.

Also, I'm placing this comment into a docstring, because it'll change in the process, so outdated comment will be properly hidden by github (thank you for attending my 20seconds of self-praising :))